zaza/openstack/charm_tests/swift/tests.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Encapsulate swift testing."""
+
+import logging
+
+import zaza.openstack.charm_tests.test_utils as test_utils
+import zaza.openstack.charm_tests.glance.setup as glance_setup
+import zaza.openstack.configure.guest
+import zaza.openstack.utilities.openstack as openstack_utils
+
+
+class SwiftImageCreateTest(test_utils.OpenStackBaseTest):
+    """Test swift proxy via glance."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running tests."""
+        super(SwiftImageCreateTest, cls).setUpClass()
+        cls.image_name = 'zaza-swift-lts'
+        swift_session = openstack_utils.get_keystone_session_from_relation(
+            'swift-proxy')
+
+        cls.swift = openstack_utils.get_swift_session_client(
+            swift_session)
+        cls.glance_client = openstack_utils.get_glance_session_client(
+            cls.keystone_session)
+
+    def test_100_create_image(self):
```

Comment:
This looks a lot like a test that exists for [Ceph RadosGW](https://github.com/openstack-charmers/zaza-openstack-tests/blob/master/zaza/openstack/charm_tests/ceph/tests.py#L580)

Reply:
Good point, I'll try and get them both using the same test,

Reply:
I'm backtracking here, but I have raised an issue saying two objects stores should share tests. However I think we should push ahead with the PR as it is based off what we currently do with amulet. 

https://github.com/openstack-charmers/zaza-openstack-tests/issues/3